.nsprc

```diff
@@ -0,0 +1,3 @@
+{
+  "exceptions": ["https://nodesecurity.io/advisories/77"]
```

Comment:
See above:
```
nsp exception for advisory 77, because request is currently blocking updating to hawk 4.1.1 and the affected modules shouldn't impact overall security for the end user
```